pyanaconda/kickstart.py

```diff
@@ -80,12 +80,32 @@
 from pykickstart.version import returnClassForVersion
 
 import logging
-log = logging.getLogger("anaconda")
+anaconda_log = logging.getLogger("anaconda")
+script_log = logging.getLogger("anaconda.kickstart.script")
+parsing_log = logging.getLogger("anaconda.kickstart.parsing")
 stderrLog = logging.getLogger("anaconda.stderr")
 storage_log = logging.getLogger("blivet")
 stdoutLog = logging.getLogger("anaconda.stdout")
 from pyanaconda import anaconda_log
 
+# command specififc loggers
+authconfig_log = logging.getLogger("anaconda.kickstart.authconfig")
```

Comment:
Is it really necessary to create a new logger for every kickstart command? Isn't there a way, how to add these prefixes, when we log the messages? We could use `log.debug("Message about a command xyz", extra={"prefix": "xyz"})` or modify the logger to be able to do something like: `log.debug_with_prefix("xyz", "Message about a command xyz")`

Example with extra arguments is here: https://docs.python.org/3/library/logging.html#logging.debug